content/sound.md

```diff
@@ -14,12 +14,64 @@
 :::
 
 
+
 ## The importance of audio
 
+- Audio is one of the most important things you can control
+- Things that can go wrong:
+  - Too quiet
+  - Too loud
+  - Instructors' volumes imbalanced
+  - Hard to hear
+  - "Ducking" (first words lower volume, or lower volume other times)
+
+
+
 ## Tips for good sound quality
 
+- Have a headset with mounted microphone
+  - Even having a good other microphone requires a room with good
+    acoustics
+  - The close pickup can't be beat with normal tools.
+  - As long as it's headset mounted, price doesn't seem to matter
+    *that* much.
+- Don't use Bluetooth
+  - Bluetooth can have too much latency (300-500ms)
+  - This may seem small but for interactive work, it's a lot
+  - Use a wired headset, or dedicated USB dongle (like gaming headsets
```

Comment:
Hmmm, what is meant by USB dongle?

Are we still talking about wired here? Or do you mean this little thing that goes into USB on laptop which the headset connects to (via Bluetooth)? Ie to not use Laptop Bluetooth directly?